### DISCUSSION TITLE
生息演算提高刷代币效率

### DISCUSSION THREAD

**Discussion by zbrocks** [MaaAssistantArknights/MaaAssistantArknights#8266] [Ideas]
发现maa在第2、3天的时候直接跳过了。由于地图是固定不变的，有个奇遇一直会存在，只是事件不固定。
其中某些事件不需要提交任何物资(多次尝试，概率还是挺高的)，进行奇遇(消耗决断)后第3天结算时能结算611代币（比之前多200）
希望开发的大佬能把这个写出来

**Comment by xmengxiao** [comment]
和我想的一样，加上这个奇遇后大概会多个十到十五秒，但是，如果遇到了要给物资换物资的选项，而刚好没有物资或者是物资不够，可能会卡在那里

**Comment by Firefly-1252** [comment]
不用点奇遇，只要在第三天的时候合成一点能量饮料然后点一个新的地点就有611代币了

**Comment by ABA2396** [comment]
自己打几天木头打完第一轮结算之后挂荧光棒速度直接翻几十倍（）
